make-prs.py

```diff
@@ -0,0 +1,251 @@
+import argparse
+import logging
+import os
+import re
+import shutil
+import subprocess
+import typing
+
+import github
+import requests
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+WORKING_DIR = "./tmp"
+DIFF_FILE = "./diff.patch"
+
+auth = github.Auth.Token(os.getenv("GITHUB_TOKEN"))
+gh_client = github.Github(auth=auth)
+
+
+def get_diff_as_patch(repo: str, pull_request_number: int):
+    logger.info("Getting diff from PR ...")
+
+    pr_diff_url = gh_client.get_repo(repo).get_pull(pull_request_number).diff_url
+
+    res = requests.get(pr_diff_url)
+    if res.status_code != requests.codes.ok:
```

Comment:
This hides the error message, I think. Maybe we should just use `res.raise_for_status()`?